app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -125,16 +155,29 @@ class HomeFragment : Fragment() {
         @JvmStatic
         fun create() = HomeFragment()
     }
+
+    fun getFocusedTilePosition(): Int {
+        return (activity.currentFocus.parent as? RecyclerView)?.getChildAdapterPosition(activity.currentFocus) ?: RecyclerView.NO_POSITION
+    }
+
+    fun dispatchKeyEvent(event: KeyEvent): Boolean {
+        if (event.keyCode == KeyEvent.KEYCODE_DEL && getFocusedTilePosition() != RecyclerView.NO_POSITION) {
```

Comment:
Why delete, is this for the emulator? If so, add a comment. Also reference the menu button behavior here.